libsolidity/CMakeLists.txt

```diff
@@ -141,6 +141,10 @@ set(sources
 	formal/SymbolicVariables.h
 	formal/VariableUsage.cpp
 	formal/VariableUsage.h
+	formal/Z3CHCSmtLib2Interface.cpp
+	formal/Z3CHCSmtLib2Interface.h
+    formal/Z3SMTLib2Interface.cpp
+    formal/Z3SMTLib2Interface.h
```

Comment:
looks like the indentation is wrong for these.

Reply:
Fixed!